### PR TITLE
Add join table for visits

### PR DESCRIPTION
**Pull request by aimalamiri**
## Add more tables
1. Add vets table
2. Add specializations table
3. Add visits table
4. Insert data into new tables
5. Run some analytical queries on it

> Here are the screenshots from my local terminal

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -122,15 +132,69 @@ INSERT INTO animals (
...
+WHERE name = 'Charmander'
+  OR name = 'Squirtle'
+  OR name = 'Blossom';
 
-UPDATE animals SET species_id = 2 WHERE name LIKE '%mon';
-UPDATE animals SET species_id = 1 WHERE name NOT LIKE '%mon';
+UPDATE animals
+SET owner_id = 5
+WHERE name = 'Angemon'
+  OR name = 'Boarmon';
```
Comment (SajjadAhmad14) [inline data.sql:179]: Please Insert the following data for specializations:
`   
 Vet William Tatcher is specialized in Pokemon.
 Vet Stephanie Mendez is specialized in Digimon and Pokemon.
 Vet Jack Harkness is specialized in
`

File: data.sql [hunk 2]
```diff
@@ -122,15 +132,69 @@ INSERT INTO animals (
...
+UPDATE animals
+SET owner_id = 2
+WHERE name = 'Gabumon'
+  OR name = 'Pikachu';
+
+UPDATE animals
+SET owner_id = 3
+WHERE name = 'Devimon'
+  OR name = 'Plantmon';
+
```
Comment (SajjadAhmad14) [inline data.sql:168]: Please Insert the following data for vets:
`    
    Vet William Tatcher is 45 years old and graduated Apr 23rd, 2000.
    Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
    Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
    Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.
`

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Aimal, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: data.sql (hunk 1 above)
Comment (aimalamiri) [reply data.sql:179]: Thanks for pointing out this part of the project. I completely forgot to add queries to my data.sql file.

**Review by Kingstalux — APPROVED** [review]
Hi @aimalamiri,
# Status: Approved ✔️ ✔️ ✔️ 
 
download


Good job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**